xds/src/main/java/io/grpc/xds/EnvoyServerProtoData.java

```diff
@@ -322,7 +325,10 @@ public String toString() {
     @Nullable
     private final SslContextProviderSupplier sslContextProviderSupplier;
 
-    FilterChain(
+    /**
+     * Creates an internal filter chain.
+     * */
```

Comment:
The prevalent style is

```
   * ....
   */
```

I am surprised the checkstyle plugin doesn't flag this when it flags many other minor things. I noticed most/all of your added Javadoc comments have this issue which could be because of the IDE. It will be nice to be consistent.